package.json

```diff
@@ -156,12 +158,13 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "node_modules/.bin/npm-run-all lint stylelint markdownlint test"
+      "pre-commit": "lint-staged"
```

Comment:
What do you think about adding `"pre-push": "npm run test"`?

Also, I'm skeptical about wanting this on every commit, because I make a lot of temporary one-off commits that I don't want to bother formatting / testing / etc and eventually squash / revert / etc.

Reply:
> What do you think about adding "pre-push": "npm run test"?

This feels similar to what we already have existing, just without the linting?

> I'm skeptical about wanting this on every commit

I think that inherently [lint-staged](https://github.com/okonet/lint-staged) only can work with pre-commit, seeing as by the time you're pushing there's nothing staged within git.

My test output seems to confirm that:

```
husky > pre-push (node v10.16.3)
i No staged files found.
```

If we want, we can table this and find an alternative solution?

Reply:
> > What do you think about adding "pre-push": "npm run test"?
> 
> This feels similar to what we already have existing, just without the linting?

Do you think we should just run tests remotely?

> > I'm skeptical about wanting this on every commit
> 
> I think that inherently [lint-staged](https://github.com/okonet/lint-staged) only can work with pre-commit, seeing as by the time you're pushing there's nothing staged within git.

I seeeeeeeee.  Ok sure, let's give this a go.  I'll make a `git commit -a -m OOPS --skip-lint-staged-junk` alias.  :P

Reply:
> > > What do you think about adding "pre-push": "npm run test"?
> > 
> > 
> > This feels similar to what we already have existing, just without the linting?
> 
> Do you think we should just run tests remotely? (Or did you want to wait until they were per-file?)

Ideally people would still run tests themselves, but maybe forcing them is a little much? I guess we could add it as well, it'd be nice to somehow be like "only if you did X changes" such that a Markdownlint change doesn't require you to have npm and python. Theoretically we could add them back to the individual file you changed if we get it working per-file?

> > > I'm skeptical about wanting this on every commit
> > 
> > 
> > I think that inherently [lint-staged](https://github.com/okonet/lint-staged) only can work with pre-commit, seeing as by the time you're pushing there's nothing staged within git.
> 
> I seeeeeeeee. Ok sure, let's give this a go. I'll make a `git commit -a -m OOPS --skip-lint-staged-junk` alias. :P

The Husky `--no-verify` flag should still work here,